File: examples-sdk/csharp/306-rag-ai-event-tracking/Main.cs
// 306-rag-ai-event-tracking — C# SDK equivalent
// Compile: vil compile --from csharp --input 306-rag-ai-event-tracking/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("customer-support-rag", 3116);
var support = new ServiceProcess("support");
support.Endpoint("POST", "/support/ask", "answer_question");
server.Service(support);
server.Compile();

File: examples-sdk/swift/306-rag-ai-event-tracking/main.swift
// 306-rag-ai-event-tracking — Swift SDK equivalent
// Compile: vil compile --from swift --input 306-rag-ai-event-tracking/main.swift --release

let server = VilServer(name: "customer-support-rag", port: 3116)
let support = ServiceProcess(name: "support")
support.endpoint(method: "POST", path: "/support/ask", handler: "answer_question")
server.service(support)
server.compile()
